Update new document of orand-v1 (#28)
orochi-network/orochi-network.github.io: src/orochi-network/orand-v1.md

# Orand V1

> Orand project was built based on Elliptic Curve Verifiable Random Function (ECVRF), it's deterministic and verifiable based on EC assumption. Orochi Network unable to manipulate the result.

Reviewer: ```suggestion
Orand project was built based on Elliptic Curve Verifiable Random Function (ECVRF). It is deterministic and verifiable based on assumptions from elliptic curves. Administrators of Orochi Network are unable to manipulate the results.
```

Is there any specific assumptions in elliptic curves? I rarely heard about the term ``EC assumption'', so I changed to assumptions in elliptic curves. It would be better if we can precisely mention those specific assumptions, like XDH, SIDH, ...

Orochi Network is a network. I am not sure whether we can say Orochi Network cannot manipulate ... Instead, can you choose some more appropriate word. For me, I am not sure so I temporarily use the word "administrators".

> Orand project was built based on Elliptic Curve Verifiable Random Function (ECVRF). It is deterministic and verifiable based on assumptions from elliptic curves. Administrators of Orochi Network are unable to manipulate the results.

Reviewer: ```suggestion
Orand project was built based on Elliptic Curve Verifiable Random Function (ECVRF). It is deterministic, verifiable and secured based on assumptions from elliptic curves. Administrators of Orochi Network are unable to manipulate the results.
```

> To optimize operation cost and improve security we provided following features:

Reviewer: ```suggestion
To optimize operation costs and improve security we provided following features:
```

> - **Verifiable:** An Orand's epoch can be verified independently outside our system or can be verified by smart contract.

Reviewer: ```suggestion
- **Verifiable:** An Orand's epoch can be verified independently outside our system or can be verified by smart contracts.
```

Is it true that a single smart contract can verify the epoch, or any smart contract (I thought of this case. So I use ``smart contracts'')

> - **Hybrid Proof System:** Our customers can choose between Fraud Proof or Validity Proof to feeding the randomness

Reviewer: ```suggestion
- **Hybrid Proof System:** Our customers can choose between Fraud Proof or Validity Proof to feed the randomness.
```

> - **Dual Proof:** An ECDSA proof will be combine with ECVRF proof to secure the feeding process, allow the feeding process to be done by any third party but the result still verifiable and immutable.

Reviewer: ```suggestion
- **Dual Proof:** An ECDSA proof will be combined with an ECVRF proof to secure the feeding process, allowing it to be done by any third party while still guaranteeing the result to be verifiable and immutable.
```

> - **Batching:** We allowed you to set the batching limit for one epoch, e.g: We can batching 1000 randomness for one single epoch. The cost will be reduced significantly.

Reviewer: ```suggestion
- **Batching:** We allow you to set the batching limit for one epoch, e.g., we can batch \\(1000\\) randomness for one single epoch which makes the cost be reduced significantly.
```

## Orand V2

> Orand V2 will focus on utilizing Multi Party Computation (MPC) to secure the generating process, allow a whole system acting as one random oracle. It will make the process more dispersity, in this stage we boot up **Chaos Theory Alliance** to preventing predictability, everything is building up toward to the vision of **Decentralized Random Number Generator**. If you believed in the vision of **Decentralized Random Number Generator** email us ([contact@orochi.network](contact@orochi.network)) to participate in **Chaos Theory Alliance**.

Reviewer: ```suggestion
Orand V2 will focus on utilizing Multi Party Computation (MPC) to secure the randomness generation, allowing the whole system to act as one random oracle. It makes the process more dispersed. In this stage, we boot up **Chaos Theory Alliance** to preventing predictability. Everything is built up toward to the vision of **Decentralized Random Number Generator**. If you believe in the vision of **Decentralized Random Number Generator**, please send drop us an email to ([contact@orochi.network](contact@orochi.network)) in order to participate in **Chaos Theory Alliance**.
```
